tests/ckeditor/basepath.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals beforeEach, describe, it, expect, CKEDITOR, window, document */
+
+'use strict';
+
+beforeEach( function() {
+	// Ensure that no CKEDITOR_BASEPATH global is available.
+	delete window.CKEDITOR_BASEPATH;
+
+	// Remove all script elements from the document.
+	removeScripts();
+} );
+
+describe( 'ckeditor.basePath', function() {
+	it( 'Full URL', function( done ) {
+		CKEDITOR.require( [ 'ckeditor' ], function( CKEDITOR ) {
+			addScript( 'http://bar.com/ckeditor/ckeditor.js' );
+			expect( CKEDITOR._getBasePath() ).equals( 'http://bar.com/ckeditor/' );
```

Comment:
I remember we've been talking about using the shortest (or reasonably short) versions. Apart from grammatical correctness, I wouldn't like to use `to.be.equal()` because it is terrible to write. Personally, I would keep `expect().equals()`.